wradlib/util.py

```diff
@@ -584,26 +585,25 @@ def has_geos():
 
 
 def get_wradlib_data_path():
-    wrl_data_path = os.environ.get("WRADLIB_DATA", None)
-    if wrl_data_path is None:
-        raise OSError(
-            "`WRADLIB_DATA` environment variable not set. Please set `WRADLIB_DATA` "
-            "pointing to the location of `wradlib_data` repository on the filesystem."
+
+    if "WRADLIB_DATA" in os.environ:
+        wrl_data_path = os.environ["WRADLIB_DATA"]
+        warn(
+            "usage of the WRADLIB_DATA environment variable will be deprecated in the future in favor of the wradlid_data package",
+            FutureWarning,
         )
```

Comment:
Usage of `WRADLIB_DATA` will not be deprecated. It will still be used, so that the user can specify the location where `wradlib-data`-package will store the data. 

We should test for availability of `wradlib-data`-package and warn if it is not yet used.

Reply:
Thanks for the review @kmuehlbauer. The comments have been integrated. You can make further changes if needed.